lib/PuppeteerSharp.Tests/DevToolsTracingTests/DevToolsTracingTests.cs

```diff
@@ -14,11 +14,11 @@
 namespace PuppeteerSharp.Tests.TracingTests
 {
     [Collection(TestConstants.TestFixtureCollectionName)]
-    public class TracingTests : PuppeteerPageBaseTest
+    public class DevToolsTracingTests : PuppeteerPageBaseTest
```

Comment:
Why the rule didn't like this name?

Reply:
The analyzer was ok with the original name on this file. I had originally renamed it to match the rename of the class under test. I'll get this changed back.